mlos_bench/mlos_bench/environments/base_environment.py

```diff
@@ -422,7 +422,7 @@ def run(self) -> Tuple[Status, datetime, Optional[Dict[str, TunableValue]]]:
 
         Returns
         -------
-        (status, timestamp, output) : (Status, datetime, dict)
+        (status, timestamp, output) : (Status, datetime.datetime, dict)
```

Comment:
does it take our imports into account? if yes, it should be just
```suggestion
        (status, timestamp, output) : (Status, datetime, dict)
```
since we do `from datetime import datetime` above in this file

Reply:
Mostly, but it's not perfect.